Examples/UICatalog/Scenarios/Editors & Helpers/AdornmentsEditor.cs

```diff
@@ -18,6 +18,8 @@ public AdornmentsEditor ()
         ExpanderButton!.Orientation = Orientation.Horizontal;
 
         Initialized += AdornmentsEditor_Initialized;
+
```

Comment:
The file path having space and ampersand in it is a bit odd. Probably not really an issue but why risk it - I'd suggest change to pascal case and use word And instead